0000-catalog.md

```diff
@@ -0,0 +1,630 @@
+- Feature Name: Catalog
+- Start Date: 2019-07-12
+- RFC PR: 
+- Hyperledger Grid Issue: [GRID-95](https://jira.hyperledger.org/browse/GRID-95)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a generic implementation for a Hyperledger Grid *Catalog*. The
+implementation of *Catalog* will materialize as a reference based ID shared between 
+instances of catalog_products. That catalog_id can be used to share a grouping of 
+products with that can be shared with one or more organizations. As well as assist 
+in programmatic operations performed on a Catalog. The Grid Catalog will contain 4 
+fields. A **catalog_id**, a **catalog_owner**, a **Name** for 
+the catalog, and a repeated field of PropertyValues for custom fields.
+
+In addition an example catalog product schema is included as a way to demonstrate 
+enforcement of additional product properties. Any grid product added to a catalog 
+will adhere to the "catalog_product" schema (or another custom-defined schema). The 
```

Comment:
Add note about attributes that go into "catalog_product" will be trading partner relationship-specific attributes like price, return policy, etc. Where the higher leve Grid Product will contain the base GS1 attributes (weights, dims, etc.)

Reply:
✅